src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,94 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITYY = 10;
+    private static final double INCREMENT_INDEX = 1.5;
+    private T[] array = (T[]) new Object[DEFAULT_CAPACITYY];
+    private int size = 0;
```

Comment:
super-redundant initializing. Java already do this for you (when you declare an int var, it has 0 value as default)
```suggestion
    private int size;
```